app/presenters/outcome_presenter.rb

```diff
@@ -1,41 +1,13 @@
 class OutcomePresenter < NodePresenter
-  class ViewContext
-    def initialize(state)
-      @state = state
-    end
-
-    def method_missing(method, *args, &block)
-      if method_can_be_delegated_to_state?(method)
-        @state.send(method, *args, &block)
-      else
-        super
-      end
-    end
-
-    def respond_to_missing?(method, include_private = false)
-      method_can_be_delegated_to_state?(method)
-    end
-
-    def get_binding
-      binding
-    end
-
-    private
-
-    def method_can_be_delegated_to_state?(method)
-      @state.respond_to?(method) && !method.to_s.end_with?('=')
-    end
-  end
-
   def initialize(i18n_prefix, node, state = nil, options = {})
     @options = options
     super(i18n_prefix, node, state)
+    @view = ActionView::Base.new(["/"])
```

Comment:
I think you mentioned not being completely happy with having to set the load path to `"/"`. Are you now satisfied that this isn't actually a problem?